src/App.js

```diff
@@ -1,23 +1,42 @@
-import logo from './logo.svg';
+
+import React,  { useEffect } from 'react'
 import './App.css';
+import { useDispatch, useSelector } from 'react-redux'
+import { loadDrinkData } from './redux/actions/liquorSearch.js'
+import { loadGameData } from './redux/actions/boardGame'
+import PairingForm from './Components/PairingForm';
+
+const App = () => {
+  const dispatch = useDispatch()
+  
+  const games = useSelector(state => state.boardGame.allGamesData.games)
+  
+  const getDrinkData = () => {
```

Comment:
I saw some conversations about moving the API calls into a separate file and I just want to say that I love that idea!